src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,29 +4,39 @@
 // </copyright>
 //---------------------------------------------------------------------
 
-using System.Diagnostics.CodeAnalysis;
+using System;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
-    public enum ODataLibraryCompatibility
+    [Flags]
+    public enum ODataLibraryCompatibility : long
     {
+        /// <summary>
+        /// No specific version or feature set.
+        /// </summary>
+        None = 0, // No flags set
+
+        /// <summary>
+        /// When enabled, the CSDL Scale and SRID "variable" value will be written "Variable" instead of "variable" for compatibility with older versions of the library.
+        /// </summary>
+        UseLegacyVariableCasing = 1 << 0,

```

Comment:
This is probably a nitpick, but I wonder whether a more specific name would be more appropriate:
```suggestion
        UsePascalVariableCasing = 1 << 0,

```
or `Uppsercase`, etc.

Cause if we ever change the casing again for some reason `Legacy` might be confusing.

Reply:
I don't see this changing since this is in protocal. I'll leave the variable name as is.